python/kwiver/vital/tests/test_camera_intrinsics.py

```diff
@@ -109,10 +104,10 @@ def test_as_matrix(self):
     def test_equal(self):
         ci1 = CameraIntrinsics()
         ci2 = CameraIntrinsics()
-        ntools.assert_true(ci1 == ci2)
-        ntools.assert_false(ci1 != ci2)
+        self.assertEqual(ci1, ci2)
+        self.assertEqual(ci1, ci2)
```

Comment:
This is the same call twice. Originally, the tests checked not only that `ci1 == ci21`, but that `ci1 != ci2` returned `False`

Reply:
Well, that's what I get for not reading carefully enough. Will update.